internal/store/deployment.go

```diff
@@ -122,6 +122,35 @@ var (
 				}
 			}),
 		},
+		// This all-in-one metric family contains all conditions for extensibility.
+		// Third party plugin may report customized condition for deployments,
+		// and Kubernetes may add new apps conditions in future.
+		{
+			Name: "kube_deployment_status_condition",
+			Type: metric.Gauge,
+			Help: "The condition of a deployment.",
```

Comment:
```suggestion
			Help: "Describes the current state of a deployment.",
```

The equivalent node metric's description isn't that great unfortunately :(. Something that needs to be fixed.

Reply:
I attempted to come to a middle ground based on the other help texts for deployments.